Keas.Mvc/Services/IdentityService.cs

```diff
@@ -199,6 +225,10 @@ public async Task<string> BulkLoadPeople(string ppsCode, string teamslug)
                     {
                         var personResult = await GetOrCreatePersonFromKerberos(kerbResults.ResponseData.Results[0].UserId, team.Id);
                         newpeople += personResult.peopleCount;
+                        if (personResult.Person == null)
+                        {
+                            warning.Append($"IAM ID {id.IamId} failed to save.");
```

Comment:
@jcubbage IAM Id isn't supper useful to me, so I'd add kerb as well, so I'd change it to:
warning.Append($"IAM ID: {id.IamId}, Kerb Id: {kerbResults.ResponseData.Results.First().UserId}  failed to save.");